packages/make-angular-cli-faster/src/utilities/make-angular-cli-faster.ts

```diff
@@ -42,4 +43,27 @@ export async function makeAngularCliFaster(args: Args) {
       'and how it can speed up your CI by up to 10 times at https://nx.dev/getting-started/nx-and-angular',
     ],
   });
+
+  if (migration.incompatibleWithAngularVersion) {
+    output.warn({
+      title:
+        'The provided version of Nx is not compatible with the currently installed version of Angular.',
+      bodyLines: [
+        'The workspace was still migrated, but you might find unexpected issues or behaviors using this version.',
+        'To know which versions are compatible, please refer to https://nx.dev/angular-nx-version-matrix.',
+      ],
+    });
+  }
+
+  // if migrated to an older version than root project is supported, log a warning
+  if (lt(migration.version, '15.3.0-beta.5')) {
+    output.warn({
+      title: 'Your workspace was not migrated to the latest version of Nx.',
```

Comment:
I'd change this to say something more along the lines of..

```suggestion
      title: 'You are not on the latest version of Nx yet.',
```